gulpfile.js

```diff
@@ -15,7 +15,7 @@ gulp.task('test', ['install-azure-account'], (cb) => {
     const env = process.env;
     env.DEBUGTELEMETRY = 1;
     env.CODE_TESTS_WORKSPACE = './test/test.code-workspace';
-    env.MOCHA_reporter = 'mocha-junit-reporter';
+    //env.MOCHA_reporter = 'mocha-junit-reporter';
```

Comment:
If you're going to comment this out then you have to set `MOCHA_reporter` somewhere else otherwise the CI builds will never fail for tests